kickstart/signup-teams.yaml

```
@@ -0,0 +1,46 @@
+summary: Sign up teams for the $SRYYYY Competition Programme
+
+priority: should
```

Can't have a competition without teams

```suggestion
priority: must
```